src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react'
+
+import { CardGroup } from './CardGroup/CardGroup'
+import { Card } from './Card/Card'
+import { Button } from '../Button/Button'
+
+export default {
+  title: 'Card',
+  parameters: {
+    info: `
+USWDS 2.0 Card components
+
+Source: https://designsystem.digital.gov/components/card/
+`,
+  },
+}
+
+const card = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Card</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }>
+    <div className="usa-card__body">
+      <p> This is a standard card with a button in the footer. </p>
+    </div>
+  </Card>
+)
+
+const cardWithMedia = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Card With Media</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }
+    media={
+      <div className="usa-card__media">
+        <div className="usa-card__img">
+          {/* Are we allowed to use this image? */}
+          <img
+            src="https://images.unsplash.com/photo-1579800070193-abe62433f737?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=610&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ"
+            alt="A placeholder"
+          />
+        </div>
+      </div>
+    }>
+    <div className="usa-card__body">
+      <p> This is a standard card with media and a button in the footer. </p>
+    </div>
+  </Card>
+)
+
+const mediaWithSetAspectRatio = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Media with Set Aspect Ratio</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }
+    media={
+      <div className="usa-card__media usa-card__media--set-aspect.add-aspect-1x1">
```

Comment:
`add-aspect-1x1` should be it's own class not part of `usa-card__media--set-aspect`.  I also saw there is a `usa-card__media--fixed-aspect` combined with add-aspect-* in the card test.

Reply:
You may want to use an image that has a larger height than width to see the effect on at responsive breakpoints.